src/runtime/rpc/rpc_host_port.cpp

```diff
@@ -157,12 +159,20 @@ std::string host_port::to_string() const
     case HOST_TYPE_IPV4:
         return fmt::format("{}:{}", _host, _port);
     case HOST_TYPE_GROUP:
-        return fmt::format("address group {}", group_host_port()->name());
+        return fmt::format("host_port group {}", group_host_port()->name());
     default:
-        return "invalid address";
+        return "invalid host_port";
     }
 }
 
+static __thread fixed_size_buffer_pool<8, 256> bf;
```

Comment:
The entire hostname, including the delimiting dots, has a maximum of 253 ASCII characters, so 256 is not enough for host and port (5 bytes at maximum)
> https://en.wikipedia.org/wiki/Hostname